server/src/main/java/org/candlepin/model/dto/ContentData.java

```diff
@@ -111,6 +111,38 @@ public ContentData() {
     }
 
     /**
+     * Initializes a new ContentData instance with the specified values.
+     * <p/></p>
+     * <strong>Note</strong>: This constructor passes the provided values to their respective
+     * mutator methods, and does not capture any exceptions they may throw due to malformed
+     * values.
+     *
+     * @param id
+     *  The ID of the content to be represented by this DTO; cannot be null
+     *
+     * @param name
+     *  The name of the content to be represented by this DTO
+     *
+     * @param type
+     *  The type of the content to be represented by this DTO
+     *
+     * @param label
+     *  The label of the content to be represented by this DTO
+     *
+     * @param vendor
+     *  The vendor of the content to be represented by this DTO
+     */
+    public ContentData(String id, String name, String type, String label, String vendor) {
```

Comment:
based on how order of the constructors in Content changes, it would be nice if this was consistent with that too